lib/index.ts

```diff
@@ -39,3 +39,4 @@ export { usePreRenderMiddleware, usePostRenderMiddleware, RenderInstruction } fr
 export { Routes, Route, IndexRoute } from './route';
 export { useTraversalMiddleware } from './match-route';
 export { QueryParams } from './query-params';
+export { LinkTo } from './link-to';
```

Comment:
In addition, I think we should provide `LinkTo` in the platform directives array like we are doing for RouteView.